app/controllers/api_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ApiController < ApplicationController
+  protect_from_forgery with: :null_session
```

Comment:
This works, but it also shuts off essentially all protection from your API controllers. I'd prefer you use the following:
```
protect_from_forgery unless: -> { request.format.json? }
```